server/routes/induction/common/qualificationDetailsController.ts

```diff
@@ -14,7 +14,10 @@ export default abstract class QualificationDetailsController extends InductionCo
     res: Response,
     next: NextFunction,
   ): Promise<void> => {
-    const { prisonerSummary, qualificationLevelForm } = req.session
+    const { pageFlowQueue, prisonerSummary, qualificationLevelForm } = req.session
+    if (!pageFlowQueue) {
```

Comment:
This check in this controller, and the next one, could (probably should) be moved out to a route request handler (the same as we do currently with `checkPrisonerSummaryExistsInSession`, `checkUpdateGoalFormExistsInSession` etc)
It's fine as it is here and now in this PR, but we should probably look to refactor it out to `routerRequestHandlers.ts` where it can be tested etc, in the future 👍